.github/workflows/scripts/linux/generate-cmake-qt.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ "${COMPILER}" = "clang" ]; then
+	echo "Using clang toolchain"
+	cat > "$HOME/clang-toolchain.cmake" << EOF
+set(CMAKE_C_COMPILER /usr/bin/clang-12)
+set(CMAKE_CXX_COMPILER /usr/bin/clang++-12)
+set(CMAKE_EXE_LINKER_FLAGS_INIT "-fuse-ld=lld")
+set(CMAKE_MODULE_LINKER_FLAGS_INIT "-fuse-ld=lld")
+set(CMAKE_SHARED_LINKER_FLAGS_INIT "-fuse-ld=lld")
+EOF
+	ADDITIONAL_CMAKE_ARGS="$ADDITIONAL_CMAKE_ARGS -DCMAKE_INTERPROCEDURAL_OPTIMIZATION=ON -DCMAKE_TOOLCHAIN_FILE=$HOME/clang-toolchain.cmake"
+fi
+
+echo "Additional CMake Args - ${ADDITIONAL_CMAKE_ARGS}"
+
+# Generate CMake into ./build
+# DISABLE_ADVANCE_SIMD is needed otherwise we end up doing -march=native.
+
+cmake \
+	-B build \
+	-G Ninja \
+	$ADDITIONAL_CMAKE_ARGS \
```

Comment:
This warning (and the others which are left) are false positives. Not sure if we can mute it specifically so it'll have a sad cross :(

Reply:
You can make codacy happy by appenging `"${ADDITIONAL_CMAKE_ARGS}"` to another value.

``` 
-DCMAKE_BUILD_TYPE=Release "${ADDITIONAL_CMAKE_ARGS}" \
```

That way it accepts the double quotes, and if the variable is empty, it will not fail.

Reply:
You broke nopch builds last time by doing exactly that

`$ADDITIONAL_CMAKE_ARGS` contains multiple arguments and we need them to be expanded as multiple arguments.  And since it's passed by the CI, we can't make an array.